Buildings/Fluid/Storage/Plant/ChillerAndTank.mo

```diff
@@ -0,0 +1,418 @@
+within Buildings.Fluid.Storage.Plant;
```

Comment:
@bravache 
Can you please clarify - Do you want me to move all controls out of `ChillerAndTank` (so that the control block will always need to be instantiated along side the model), or just package them into one block like what I am currently doing with `.BaseClasses.ReversiblePumpValveControl`?

Reply:
This should be addressed in ff5d9ca, eea6111, and 05dcef4.